### ISSUE TITLE
Double tap + drag to zoom in/out

### ISSUE THREAD

**Issue by flavioarfaria** [cyrilmottier/Polaris#17]
Recently, Google Maps app added a feature that allows users to tap the map twice and drag the finger (keeping the finger on screen after the second tap) to zoom in/out. It enables users to use the map app with one hand only, since pinch-to-zoom is not a straightforward gesture to be performed when using only one hand.

**Comment by cyrilmottier** [comment]
Yes, I know about this new gesture. I would love to have it in Polaris. Unfortunately, technical limitations makes it impossible. In order to do so we would need to be able to zoom in/out in at non-discrete levels, which is impossible using `MapView`.

If you have a solution I'd be happy to hear it. Otherwise, I will close no choice but to close this issue as "Not implementable" :(.

**Comment by flavioarfaria** [comment]
I also know about this limitation, but I was considering having it working the same way pinch to zoom does. The more the user moves his finger away of the tap point, the more the value of the discrete zoom changes. Do you think it's a bad idea?

**Comment by cyrilmottier** [comment]
If you can have it working like it does in the Maps application, I would be pleased to merge your pull request :).
